staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -12,12 +14,54 @@
 
 class PlasmidfinderBlastDatabase(AbstractBlastDatabase):
 
-    def __init__(self, database_dir: str) -> None:
+    def __init__(self, database_dir: str, bacteria=None) -> None:
         """
         Creates a new PlasmidfinderBlastDatabase.
-        :param database_dir: The specific ResFinder database (drug class) directory.
+        :param database_dir: The specific PlasmidFinder database directory.
         """
+
         super().__init__(database_dir)
+        self.plasmidfinder_database_dir = path.join(self.database_dir)
+        self.bacteria = bacteria
+
+        if bacteria is not None:
+            self.plasmidfinder_database_file = path.join(self.database_dir, bacteria)
+            self.plasmidfinder_database_file += ".fsa"
+
+            if (not path.isfile(self.plasmidfinder_database_file)):
+                raise Exception(
+                    "Error, plasmidfinder bacteria [" + bacteria + "] is either incorrect or plasmidfinder database not installed properly")
```

Comment:
You could also use this syntax to insert variables into strings if you want:

```python
raise Exception("Error, plasmidfinder bacteria [{}] is either incorrect or plasmidfinder database not installed properly", bacteria)
```

There's also this syntax:

```python
raise Exception(f"Error, plasmidfinder bacteria [{bacteria}] is either incorrect or plasmidfinder database not installed properly")
```

But it's only in [Python 3.6](https://docs.python.org/3.7/whatsnew/3.6.html?highlight=interpolation#pep-498-formatted-string-literals) and above (I was trying to keep staramr runnable in Python 3.5).

Reply:
I went with the first suggestion you mentioned to keep consistent with similar syntax in the codebase. Resolved in commit dbfcea1